packages/type/scss/_reset.scss

```
@@ -75,7 +75,13 @@
   }
 
   a {
-    color: #0062ff;
+    @if global-variable-exists('carbon--theme') and
+      map-has-key($carbon--theme, 'link-01')
+    {
+      color: map-get($carbon--theme, 'link-01');
+    } @else {
+      color: #0062ff;
```

Based on the snapshot I think we want this to be #0062fe 

```suggestion
      color: #0062fe;
```

---

thanks for the help guys